packages/payment-detection/src/erc20/proxy-info-retriever.ts

```diff
@@ -64,8 +64,12 @@ export default class ProxyERC20InfoRetriever
 
   /**
    * Retrieves transfer events for the current contract, address and network.
+   * @param isReceivable Whether or not this proxy contract is a receivable contract, which
+   * ignores the toAddress filtering since they can be transferred to other recipients.
    */
-  public async getTransferEvents(): Promise<PaymentTypes.ERC20PaymentNetworkEvent[]> {
+  public async getTransferEvents(
+    isReceivable = false,
```

Comment:
I'd like to get @benjlevesque 's opinion on this. I was first against overriding the erc20 proxy info retriever, but knowing that a transferrable receivable can be paid through the usual proxy leads me to accept that solution.